cdap-common/src/main/java/co/cask/cdap/common/twill/MasterServiceManager.java

```diff
@@ -86,5 +86,20 @@
 
   SystemServiceLiveInfo getLiveInfo();
 
-  //TODO: Add method to get the metrics name to get event rate on UI
```

Comment:
Ah no, this is due to some mistake I made commit to my local develop branch. Ignore this one =)